webapp/channels/src/components/convert_channel_modal/convert_channel_modal.tsx

```diff
@@ -84,26 +82,28 @@ export default class ConvertChannelModal extends React.PureComponent<Props, Stat
                 </Modal.Header>
                 <Modal.Body>
                     <p>
-                        <FormattedMarkdownMessage
-                            id='convert_channel.question1'
-                            defaultMessage='When you convert **{display_name}** to a private channel, history and membership are preserved. Publicly shared files remain accessible to anyone with the link. Membership in a private channel is by invitation only.'
+                        <FormattedMessage
+                            id='convert_channel.question_1'
```

Comment:
```suggestion
                            id='convertChannel.question1'
```

Same pattern for others as well